mep/people/management/commands/export_creators.py

```diff
@@ -0,0 +1,40 @@
+"""
+Manage command to export creator data for use by others.
+
+Generates a CSV and JSON file including details on every creator
+(author, translated, editor, etc) in the database, with details
+on creator nationality, gender, and other information.
+"""
+
+from mep.people.models import Person
+from mep.people.management.commands.export_members import Command as ExportMemberCommand
+
+
+class Command(ExportMemberCommand):
+    """Export creator data."""
+
+    csv_fields = [
+        "id",  # no URI for authors so using slug as ID
+        "name",
+        "sort_name",
+        "title",
+        "gender",
+        "is_organization",
+        "birth_year",
+        "death_year",
+        "viaf_url",
+        "wikipedia_url",
+        # related country
+        "nationalities",
+        # generic
+        "notes",
+        "updated",
+    ]
+
```

Comment:
In #684 Josh requested/suggested creator types and associated items - creator types seems like it would be useful. I don't know if you already have a solution for associating books and creators.